package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "startwords",
+  "version": "1.0.0",
+  "author": "The Center for Digital Humanities at Princeton <cdhdevteam@princeton.edu>",
+  "description": "",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
since `private = true` and we'll never publish the package, you're free to take out the generated `repository`, `scripts`, `bugs`, `homepage`, `description`, etc. keys if you want. I think npm/yarn only check for the presence of `dependencies` and `devDependencies`, though they might warn about other things missing.